apps/web/src/component/Onboarding/PhoneMask.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactElement, forwardRef } from "react";
+import { IMaskInput } from "react-imask";
+interface PhoneMaskProps {
+  onChange: (event: { target: { name: string; value: unknown } }) => void;
+  name: string;
+}
+
+export const PhoneMask = forwardRef<ReactElement, PhoneMaskProps>(
+  function PhoneMask(props, ref) {
+    const { onChange, ...other } = props;
+    return (
+      <IMaskInput
+        {...other}
+        mask="+AB 000 000 0000"
+        definitions={{
+          A: /[6]/,
+          B: /[1,4]/,
+        }}
+        ref={ref}
```

Comment:
```suggestion
        inputRef={ref as RefCallback<HTMLTextAreaElement | HTMLInputElement>}
```

this will remove the warning that you have right now in the test and console related to:
```
    console.error
      MUI: You have provided a `inputComponent` to the input component
      that does not correctly handle the `ref` prop.
      Make sure the `ref` prop is called with a HTMLInputElement.
```